ui/src/components/NodePageMetricsTab.js

```diff
@@ -406,7 +404,7 @@ const NodePageMetricsTab = (props) => {
                 lineConfig={lineConfig}
               />
             ) : (
-              <NoDataGraphText>Loading…</NoDataGraphText>
+              <LoaderContainer size="small"></LoaderContainer>
```

Comment:
Using a loader instead of some text isn't related to the "code deduplication" commit, would be better to create another one.